### ISSUE TITLE
Add an option to include external libraries directly.

### ISSUE THREAD

**Issue by hkirat** [mozilla/thimble.mozilla.org#2347]
Not sure if there has already been a discussion on this.
How would it be to include a feature that allows users to load external libraries, without including them in the source code.
Eg: If a developer wants to learn aframe, we could include aframe.js and he can get to development directly

We would provide an option for famous libraries like jquery, processing(p5.js) etc.

While exporting the program, we could probably inject the scripsts in the head of the index.html file.

**Comment by humphd** [comment]
We've discussed this before, yes, and there are differing views on the right approach.  Some people think we should go this route, and others think you should provide pre-baked projects you can fork that have what you want already included/setup.

Thimble is about projects of files, so if you want to build a Bootstrap project, AFrame project, etc we should provide one that you can remix and get started using.  That's my opinion.  It's not universally held, though.

**Comment by hkirat** [comment]
Yes that makes sense. However I think that the showcase on Thimble's homepage is pretty restricted (6 projects to remix from). 
What if we could add a button somewhere in the topbar as a dropdown which lists some of the common library's starter projects.
cc @flukeout
